NeuralAmpModeler/scripts/makedist-win.bat

```diff
@@ -1,142 +1,144 @@
-@echo off
```

Comment:
Why the humongous diff?

It looks like things weren't actually changed, but not being able to see the "actual" change makes this difficult to review from my phone 😉 (and isn't desirable in the future for making the commit history easy to understand). Any way you can fix this?

Reply:
Hmmm, sorry... totally my bad.  LOL @ reviewing changes from a phone, that's crazy hard albiet very dedicated of you.

It also reminded me that I never actually achieved my goal of compiling a MacOS 10.9 compatible version, so I will get back to that.